client/my-sites/plan-features-2023-grid/components/plan-features.tsx

```diff
@@ -0,0 +1,105 @@
+import { JetpackLogo } from '@automattic/components';
+import { useDesktopBreakpoint } from '@automattic/viewport-react';
+import { LocalizeProps } from 'i18n-calypso';
+import { useState, useEffect } from 'react';
+import PlanFeatures2023GridFeatures from '../components/features';
+import { PlanProperties, DataResponse } from '../types';
+import { Plans2023Tooltip } from './plans-2023-tooltip';
+import type { DomainSuggestion } from '@automattic/data-stores';
+
+const PlanFeatures: React.FC< {
+	planProperties: PlanProperties[];
+	paidDomainName?: string;
+	wpcomFreeDomainSuggestion: DataResponse< DomainSuggestion >; // used to show a wpcom free domain in the Free plan column when a paid domain is picked.
+	translate: LocalizeProps[ 'translate' ];
+	hideUnavailableFeatures?: boolean; // used to hide features that are not available, instead of strike-through as explained in #76206
+	selectedFeature?: string;
+	isCustomDomainAllowedOnFreePlan: DataResponse< boolean >; // indicate when a custom domain is allowed to be used with the Free plan.
+	isTableCell: boolean | undefined;
+	Container: (
+		props: React.HTMLAttributes< HTMLDivElement > | React.HTMLAttributes< HTMLTableCellElement >
+	) => JSX.Element;
+} > = ( {
+	planProperties,
+	paidDomainName,
+	wpcomFreeDomainSuggestion,
+	translate,
+	hideUnavailableFeatures,
+	selectedFeature,
+	isCustomDomainAllowedOnFreePlan,
+	isTableCell,
+	Container,
+} ) => {
+	const [ activeTooltipId, setActiveTooltipId ] = useState( '' );
+	const isDesktop = useDesktopBreakpoint();
```

Comment:
The plans grid uses different breakpoints from the rest of Calypso, so I think using `useDesktopBreakpoint` would cause issues at certain breakpoints:
https://github.com/Automattic/wp-calypso/blob/178ed82ccf2a0b094e6d60d0526184d378126a36/client/my-sites/plan-features-2023-grid/_media-queries.scss#L75-L80
https://github.com/Automattic/wp-calypso/blob/8d3ad9ee07de9d7e55ae2be42c5b3e26069e535d/client/my-sites/plan-features-2023-grid/_media-queries.scss#L6-L8

We're figuring out ways to improve this, here's the WIP PR: https://github.com/Automattic/wp-calypso/pull/79129.

I don't have any good solutions to move this PR forward, maybe something like `isDesktop = useBreakpoint('>780px')`?

Reply:
I realised that I don't need the breakpoints at all. I was using it to determine whether or not to add the touch event listeners, and I made a change to check for whether the browser supports touch events instead of relying on viewport size.